woudc_data_registry/product/uv_index/uv_index_generator.py

```diff
@@ -182,32 +189,36 @@ def execute(path, update, start_year, end_year, bypass):
                     except Exception as err:
                         msg = f'Unable to compute UV for file {ipath}: {err}'  # noqa
                         LOGGER.error(msg)
+                        unsuccessful_files.append(ipath)
+                        continue
 
-                    # form ids for data insert
-                    contributor_id = ':'.join([agency, project_id])
-                    deployment_id = ':'.join([station_id, contributor_id])
-                    instrument_id = ':'.join([instrument_name,
-                                              instrument_model,
-                                              instrument_number, dataset_id,
-                                              deployment_id])
-                    # check if instrument is in registry
-                    exists = registry_.query_by_field(Instrument,
-                                                      'instrument_id',
-                                                      instrument_id)
-                    if not exists:
-                        # instrument not found. add it to registry
-                        if bypass:
-                            LOGGER.info('Skipping instrument addition check')
-                            allow_add_instrument = True
-                        else:
-                            response = \
-                                input(f'Instrument {instrument_id} not found. '
-                                      'Add? (y/n) [n]: ')
-                            allow_add_instrument = \
-                                response.lower() in ['y', 'yes']
-
-                        if allow_add_instrument:
-                            instrument_ = {
+                # form ids for data insert
+                contributor_id = ':'.join([agency, project_id])
+                deployment_id = ':'.join([station_id, contributor_id])
+                instrument_id = ':'.join([instrument_name,
+                                          instrument_model,
+                                          instrument_number, dataset,
```

Comment:
Correction: should be `dataset_id` here.
```suggestion
                                          instrument_number, dataset_id,
```